src/TodoList.jsx

```diff
@@ -1,8 +1,8 @@
 import TodoListItem from './TodoListItem.jsx'
 
-const TodoList = ({todoList}) => (
+const TodoList = ({todoList, onRemoveTodo}) => (
   <ul>
-    {todoList.map(item => ( <TodoListItem key={item.id} item={item} /> ))}
+    {todoList.map(item => ( <TodoListItem key={item.id} item={item} onRemoveTodo={onRemoveTodo} /> ))}
```

Comment:
I would suggest formatting here like this - this is a common practice to follow: 

```
  <ul>
    {todoList.map((item) => (
      <TodoListItem key={item.id} item={item} onRemoveTodo={onRemoveTodo} />
    ))}
  </ul>
);

```